test/parallel/test-buffer-tostring-ascii.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+// Flags: --expose-gc
+
+const common = require('../common');
+const assert = require('assert');
+
+// Maximum length in bytes that V8 can handle.
+// This equals to 268435440 bytes = (256 * 1024 * 1024) - 16
+const kMaxLength = (1 << 28) - 16;
+
+const skipMessage =
+  '1..0 # Skipped: intensive toString tests due to memory confinements';
+if (!common.enoughTestMem) {
+  console.log(skipMessage);
+  return;
+}
+assert(typeof gc === 'function', 'Run this test with --expose-gc');
+
+try {
```

Comment:
Should use an appropriate `assert.throws` or `assert.doesNotThrow` here instead of throwing the raw error